Project-Lavinia/Lavinia-api
# Added a request for available election years to the API

**Changes:**
- Added a way of requesting the available election years from the API
- Renamed the database: ModelDB -> NODatabase to match NOContext and NOInitializer
- Renamed the Swagger page title: Swagger UI -> Lavinia API
- Updated Swagger: 3.0.0 -> 3.0.1
- Updated xUnit: 2.4.0 -> 2.4.1

**Motivation:**
- First three changes are explained in the related issues #19, #20 and #17 
- The new swagger version utilises .Net Core 2 features for performance gains
- The new xUnit version comes with improved speed

**Setup:**
Make sure to run `npm restore` to get the updated packages.

**Testing:**
- Ensure the API returns expected results because of the db rename and Swagger update
- Ensure the API returns all available years in sorted from newest to oldest
- Check that the Swagger page is titled "Lavinia API"
- Run all the tests to validate the latest version of xUnit

**Issues:**
Closes #17 
Closes #19 
Closes #20
